### PR TITLE
fix: bulk e-waybill print

### PR DESCRIPTION
**Pull request by ljain112**
Continued : #1243

### CODE REVIEW

File: india_compliance/gst_india/client_scripts/sales_invoice_list.js [hunk 1]
```diff
@@ -76,6 +84,36 @@ function show_bulk_update_transporter_dialog(docnames) {
     d.show();
 }
 
+async function bulk_e_waybill_print(docnames) {
+    const json_string = JSON.stringify(docnames);
+
+    const response = await fetch(
+        `/api/method/india_compliance.gst_india.utils.e_waybill.bulk_e_waybill_print?` +
```
Comment (rtdany10) [inline india_compliance/gst_india/client_scripts/sales_invoice_list.js]: Why not open this URL in a new window?

Comment (ljain112) [reply india_compliance/gst_india/client_scripts/sales_invoice_list.js]: done

File: india_compliance/gst_india/client_scripts/sales_invoice_list.js [hunk 2]
```diff
@@ -76,6 +84,38 @@ function show_bulk_update_transporter_dialog(docnames) {
...
+                        )
+                    );
+                }
+
+                window.open_url_post(
+                    "/api/method/frappe.utils.print_format.download_multi_pdf",
+                    {
+                        doctype: "e-Waybill Log",
+                        name: JSON.stringify(r.message.valid_log),
+                        format: "e-Waybill",
```
Comment (rtdany10) [inline india_compliance/gst_india/client_scripts/sales_invoice_list.js]: Let's not hardcode the print format, let system take the standard one

**Comment by rtdany10** [comment]
LGTM, thank you!

**Review by vorasmit — APPROVED** [review]

**Pull request merged**